docs/pages/access-controls/sso/okta.mdx

```diff
@@ -195,11 +195,11 @@ $ tctl create dev.yaml
 ## Testing
 
 The Web UI now contains a new "Okta" button at the login screen. The CLI
-behaves the same as before, but now you provide the email associated with an
-Okta account:
+allows you to just specify the Proxy address and it will automatically
```

Comment:
To me, this sounds like it's a feature of the CLI that you can specify the Proxy Service address. How about (I can't suggest because of the deleted lines):

```markdown
The Web UI now contains a new "Okta" button at the login screen. To 
authenticate via the `tsh` CLI, specify the Proxy Service address and `tsh` will 
automatically use the default SSO type:
```

Reply:
What is a _default SSO type_?

Reply:
The default 

> What is a _default SSO type_?

The default authentication type (local, saml, github, oidc,...).

Reply:
Please see update.